ddtrace/appsec/iast/loader.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+
+from ddtrace.internal.logger import get_logger
+from ddtrace.internal.module import ModuleWatchdog
+
+from ._ast.ast_patching import _should_iast_patch
+from ._ast.ast_patching import astpatch_source
+from ._util import _is_iast_enabled
+
+
+log = get_logger(__name__)
+
+
+IS_IAST_ENABLED = _is_iast_enabled()
+
+
+def _exec_iast_patched_module(module_watchdog, module):
+    patched_source = None
+    if IS_IAST_ENABLED:
+        log.debug("IAST enabled")
+        module_path, patched_source = astpatch_source(module.__name__, None)
```

Comment:
if we pass just the module, e.g.
```suggestion
        module_path, patched_source = astpatch_source(module)
```
then we might be able to use `ddtrace.internal.module.origin` to retrieve the module path and avoid the code for getting the spec and then the origin in `ast_patching.py`?

Reply:
Thanks for the idea! I prefer keeping this signature because it's easier to test (it just needs the name and path of the modules, and not a Module object), and set up an intermediate function (``astpatch_module``) to retrieve the module origin as you suggest and pass it to this one